pyth-sdk-terra/README.md

```diff
@@ -104,3 +104,7 @@ List of available Price Feeds and their ids:
 
 #### Notes
 - :warning: `num_publishers` and `max_num_publishers` in `PriceFeed` are currrently unavailable and set to 0. 
+
+## Examples
+
+Please checkout [Example Terra Contract](../examples/terra-contract/) as an example which uses Pyth Terra contract to fetch a price. This example also includes guidance for how to [write](../examples/terra-contract/Developing.md), and [deploy and query](../examples/terra-contract/tools/README.md) contracts.
```

Comment:
I would actually put this under the Usage section above. This is very important / relevant info for most people, so it deserves to be early in the document.

Reply:
Yes. I agree.